microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md

```diff
@@ -88,14 +88,17 @@ Let's say you've deleted a user account and now you want to convert their old ma
 
 ## Convert a user's mailbox in a hybrid environment
 
-If this shared mailbox is in a hybrid environment, we **strongly recommend** (almost require!) that you move the user mailbox back to on-premises, convert the user mailbox to a shared mailbox, and then move the shared mailbox back to the cloud. 
+Since October 11th, 2018, Exchange Hybrid now supports creating remote shared mailboxes starting in Cumulative Update 21 for Exchange Server 2013 and Cumulative Update 10 for Exchange Server 2016 in an on-premises Exchange Server environment, you can directly create or modify a remote shared mailbox by using the new parameter -shared. For mroe information, Visit [Cmdlets to create or modify a remote shared mailbox in an on-premises Exchange environment](https://support.microsoft.com/en-us/help/4133605/cmdlets-to-create-modify-remote-shared-mailbox-in-on-premises-exchange)
```

Comment:
```suggestion
Since October 11th, 2018, Exchange hybrid deployment supports creating remote shared mailboxes starting in Cumulative Update 21 for Exchange Server 2013 and Cumulative Update 10 for Exchange Server 2016 in an on-premises Exchange Server environment. You can directly create or modify a remote shared mailbox by using the new -shared parameter. For more information, visit [Cmdlets to create or modify a remote shared mailbox in an on-premises Exchange environment](https://support.microsoft.com/help/4133605/cmdlets-to-create-modify-remote-shared-mailbox-in-on-premises-exchange).
```